gradlew

```diff
@@ -1,54 +1,64 @@
 #!/usr/bin/env bash
+# Created by unknown under unknown
+# Refactored by Jacob Hrbek <kreyren@rixotstudio.cz> under original license in 2020
 
-##############################################################################
-##
-##  Gradle start up script for UN*X
-##
-##############################################################################
+: "
+  Gradle start up script for UN*X
+
+  FIXME: Abstract
+"
+
+# Master unset -- Used to ensure that script does not export unwanted env variables
+masterUnset="DEFAULT_JVM_OPTS APP_NAME APP_BASE_NAME MAX_FD cygwin msys darwin cygwin darwin msys PRG ls link PRG PRG SAVED APP_HOME CLASSPATH JAVACMD JAVACMD JAVACMD MAX_FD_LIMIT MAX_FD GRADLE_OPTS name icon APP_HOME CLASSPATH JAVACMD ROOTDIRSRAW SEP ROOTDIRS SEP OURCYGPATTERN OURCYGPATTERN CHECK CHECK2 JVM_OPTS appname masterUnset"
+
+# Functions
+ewarn() { printf 'WARN: %s\n' "$1" ;}
+
+die() {
+    printf 'FATAL: %s\n' "$2"
+
+    # masterUnset - Do not double-quote since spaces are expected
+    for envVar in $masterUnset; do
```

Comment:
Sorry, you're running shellcheck, but then adding code that uses word-splitting of an unquoted var, causing a failure of the shellcheck you've just added?